README.md

```diff
@@ -90,6 +90,12 @@ Refer to our [API documentation](http://docs.agentops.ai) for detailed instructi
 |🔜 Information context constraint testing | 🔜 Anti-agent roadblocks (i.e. Captchas) | 🔜 CI/CD integration checks | |
 |🔜 Regression testing | 🔜 Multi-agent framework visualization | | |
 
+## Integrations
+
+### CrewAI 🛶
+
+Write Crew agents with observability with only 2 lines of code. [More information here.](https://docs.crewai.com/how-to/AgentOps-Observability/). Simply set an `AGENTOPS_API_KEY` in your environment, and your crews will automatically be set up with.
```

Comment:
Should we point to our own docs for Crew? That way, we have control and can update the docs more easily? Our docs can then point to their docs

Reply:
Good call. I'd actually opt in for both since we have greater control of our own docs, but it also shows we have an official integration by linking to theirs.

Regardless, would like to keep a snippet in the readme because a lot of crew users are adding AgentOps